src/components/shared/Promotions/Promotions.js

```diff
@@ -39,7 +39,7 @@ const Promotions = function({ intl, type, page, dependencies: { dopplerSitesClie
 
       fetchData();
     }
-  }, []);
+  }, [defaultBannerData, dopplerSitesClient, page, intl, type]);
```

Comment:
Oh, since `defaultBannerData` is a dependency, you should move it outside of _render_ scope, in another case, _the effect_ will be executed each time that this component is rendered.

Let me know if I am not being clear enough.

Reply:
fixed